src/events/components/DetailView/index.tsx

```diff
@@ -23,15 +24,17 @@ export const DetailView = ({ eventId }: IProps) => {
   const isPending = useSelector((state) => state.events.loading === 'pending');
 
   useEffect(() => {
+    window.scrollTo(0, 0);
```

Comment:
Why is scrolling to the top needed?

Reply:
It is for scrolling to the top of the page when the displayed event is changed. Lets say you go from one event page to another. Keeping the scroll position wouldn't make sense for the user. But react doesn't know that, it just sees a prop change

Kinda just copied it from careeropportunities, since I remember we added it there after an issue was filed on it

Reply:
Oh, right! So Next doesn't render another "page" and rerenders when the url is changed?